user/dnscrypt-proxy/patches/dnscrypt-proxy-toml.patch

```diff
@@ -0,0 +1,329 @@
+diff --git i/dnscrypt-proxy/example-dnscrypt-proxy.toml w/dnscrypt-proxy/example-dnscrypt-proxy.toml
```

Comment:
if you're refactoring things make it so that config file isn't installed in `/etc`

Reply:
The `template.py` file doesn't install anything inside `/etc`. The patch mentions the default location of config files as `/etc/dnscrypt-proxy` only in the comments.

Reply:
Should I change 'etc' to some location in 'usr' in the comments as well?